docker-compose.yml

```diff
@@ -4,6 +4,13 @@ redis:
   image: redis:2.8
 rabbitmq:
   image: rabbitmq:3.5
+solr:
+  image: makuk66/docker-solr:4.10.4
+  volumes:
+    # hydroshare repository
+    - "/home/hydro/hydroshare:/hydroshare"
```

Comment:
hydroshare repository is volume-mapped to /home/docker/hydroshare for hydroshare container, but mapped to /hydroshare here for solr container, which does not sound right to me. Also, it is probably best to use '.' rather than hardcoded /home/hydro/hydroshare since people may not use the hydro vm for their development environment.